include/ublksrv_priv.h

```diff
@@ -81,10 +81,8 @@ static inline void *ublksrv_get_sqe_cmd(struct io_uring_sqe *sqe)
 
 static inline void ublksrv_set_sqe_cmd_op(struct io_uring_sqe *sqe, __u32 cmd_op)
 {
-	__u32 *addr = (__u32 *)&sqe->off;
-
-	addr[0] = cmd_op;
-	addr[1] = 0;
+	sqe->cmd_op = cmd_op;
```

Comment:
When I write ublksrv, the header isn't updated yet, even though SQE128 is supported. It is just fine to use the current way, at least it still can build successfully against old header.

But your patch is fine, and I will apply it after automake patches are merged, 
and liburing2.2 is required explicitly by that patchset.

Reply:
Sure! I didn't know about the history and didn't check liburing versions. I'm just going through the code to understand how it works and how I can use it for fuse.

Reply:
BTW, just curious how you use it for fuse?  I understand fuse is userspace FS, and look it needn't one block device.